worker-sys/src/types/hyperdrive.rs

```diff
@@ -0,0 +1,26 @@
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
+    #[wasm_bindgen(extends=js_sys::Object)]
+    #[derive(Clone, PartialEq, Eq)]
+    pub type Hyperdrive;
+
+    #[wasm_bindgen(method, getter, js_name=connectionString)]
+    pub fn connection_string(this: &Hyperdrive) -> String;
```

Comment:
Open questions
1. Is `method` the correct attribute?
2.  Are these property accessors fallible? Should these return a `Result`?

Reply:
I do believe that `method` is correct. I don't think these attributes could fail, although I suppose it couldn't hurt to catch it.